src/test/regress/output/multi_copy.source

```diff
@@ -25,24 +25,24 @@ ERROR:  could not find any shards into which to copy
 DETAIL:  No shards exist for distributed table "customer_copy_hash".
 HINT:  Run master_create_worker_shards to create shards and try again.
 SELECT master_create_worker_shards('customer_copy_hash', 64, 1);
- master_create_worker_shards 
------------------------------
- 
+ master_create_worker_shards
+---------------------------------------------------------------------
+
 (1 row)
 
 -- Test empty copy
 COPY customer_copy_hash FROM STDIN;
 -- Test syntax error
 COPY customer_copy_hash (c_custkey,c_name) FROM STDIN;
-ERROR:  invalid input syntax for type integer: "1,customer1"
+ERROR:  invalid input syntax for integer: "1,customer1"
```

Comment:
why does this error change?

Reply:
I think it changed in a major Postgres version, and the output differences are ignored.